src/stcal/jump/jump.py

```diff
@@ -235,13 +245,13 @@ def detect_jumps(
 
     # Apply gain to the SCI, ERR, and readnoise arrays so they're in units
     # of electrons
-    data *= gain_2d
-    err *= gain_2d
+    data = indata * gain_2d
+    err = inerr * gain_2d
     readnoise_2d *= gain_2d
     # also apply to the after_jump thresholds
     after_jump_flag_e1 = after_jump_flag_dn1 * np.nanmedian(gain_2d)
     after_jump_flag_e2 = after_jump_flag_dn2 * np.nanmedian(gain_2d)
-
+    print("after_jump_flag_e1: shape", after_jump_flag_e1.shape)
```

Comment:
No print statements in production code. Either remove or change to a log message.

Reply:
removed print